runtime/doc/options.txt

```diff
@@ -4166,6 +4166,26 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	The 'mousemodel' option is set by the |:behave| command.
 
+					*mousescroll*
+'mousescroll'		string	(default "ver:3,hor:6")
+			global
+	This option controls the number of lines / columns to scroll by when
+	scrolling with a mouse. The option is a comma separated list of parts.
+	Each part consists of a direction and a count as follows:
+		direction:count,direction:count
+	Direction is one of either 'hor' or 'ver', 'hor' controls horizontal
+	scrolling and 'ver' controls vertical scrolling. Count sets the amount
```

Comment:
Double quotes should be used here:
```suggestion
	Direction is one of either "hor" or "ver", "hor" controls horizontal
	scrolling and "ver" controls vertical scrolling. Count sets the amount
```

Reply:
Made the change. Squashed the commit for cleaner repo history. Thanks for reviewing these old PRs btw 🙂.